### DISCUSSION TITLE
广州本地公共交通优惠政策改变

### DISCUSSION THREAD

**Discussion by ashley-suen** [Ivysauro/CNRT#66] [General]
8月25日，广州地铁集团与广州公交集团联合发布《关于实施〈广州市公共交通票价优惠调整方案〉的公告》（下称“公告”）。根据公告，《广州市公共交通票价优惠调整方案》将于9月1日起正式实施，优惠方案由原来的“累次优惠”调整为“累额优惠”，新增低收入居民5折乘车、公共交通乘车码同享优惠措施，并将佛山地铁已开通线路纳入累额计算中。
　　具体而言，新的票价优惠方案由原来的“满15次六折”，调整为“普通乘客在一个自然月内，乘坐广州地铁公交累计实际支出票款满80元不满200元部分享受8折优惠；超出200元部分享受5折优惠”，即多乘优惠更多，公交、地铁、水巴同享优惠。

**Comment by chenw23** [comment]
已复核并纳入 PR #153。广州自 2023-09-01 起已从“95折、15次后6折”改为累额优惠：自然月累计实际支付满80元不满200元部分8折，超过200元部分5折；公共交通乘车码纳入优惠，同一卡或同一乘车码入口累计；善行卡适用低收入居民5折。

PR：https://github.com/Ivysauro/CNRT/pull/153
依据：https://www.gz.gov.cn/zwfw/zxfw/jtfw/content/mpost_9178409.html
